src/core/searchUtils.js

```diff
@@ -92,6 +92,22 @@ export function loadSearchResultsIfNeeded({ store: { dispatch, getState }, locat
     ...location.query,
     clientApp: state.api.clientApp,
   });
+
+  if (!isLoaded({ state: state.search, page, filters })) {
+    return performSearch({ dispatch, page, api: state.api, auth: state.auth, filters });
+  }
+  return true;
```

Comment:
I agree actually, as it makes testing weird too. That said:

![it-was-like-that-when-i-got-here](https://cloud.githubusercontent.com/assets/90871/20501664/ad9ac306-b032-11e6-8f7e-00ffb3e6ddca.jpg)

I've filed an issue for it (#1403), as changing this across the app would be out-of-scope for the PR.